workflow-dev/tpl/deis-controller-rc.yaml

```diff
@@ -36,6 +36,8 @@ spec:
             - containerPort: 8000
               name: http
           env:
+            - name: KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS
+              value: 5
```

Comment:
When running locally, I get the following on installation of chart: 

`unable to decode "STDIN": [pos 860]: json: expect char '"' but got char '5'`

Thinking this needs to read `value: "5"`